arelle/plugin/validate/ESEF_2022/DTS.py

```diff
@@ -110,7 +111,9 @@ def lc3wordAdjust(word: str) -> str:
                         if modelConcept not in val.primaryItems:
                             extLineItemsWithoutHypercube.append(modelConcept)
                         elif not widerNarrowerRelSet.fromModelObject(modelConcept) and not widerNarrowerRelSet.toModelObject(modelConcept):
-                            if not calcRelSet.fromModelObject(modelConcept): # exclude subtotals
+                            # Reporting manual - 1.4 Anchoring -> RTS on ESEF does not set an anchoring requirement for the Notes
+                            # to the financial statements
+                            if not calcRelSet.fromModelObject(modelConcept) and not isAnchoredToNotes(modelConcept, parentChildRelSet, set()): # exclude subtotals
```

Comment:
The condition reads somewhat confusingly as "anchoring not required if anchored to notes".  Perhaps `isChildOfNotes` would be a clearer name?

Reply:
Yes, we shall change the name from `isAnchoredToNotes` to `isChildOfNotes`.